src/components/periodic-table/PeriodicTable.test.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
+import PeriodicTable, { IPeriodicTableElement } from "./PeriodicTable";
+import {
+  render,
+  screen,
+  waitForElementToBeRemoved,
+} from "@testing-library/react";
+import PtElementInfo from "../pt-element/PtElementInfo";
+import ElementManager from "../../ElementManager";
+
+function elementRenderer(atomic: number): IPeriodicTableElement {
+  return {
+    // @ts-ignore fix this
+    component: PtElementInfo,
+    props: {
+      element: ElementManager.getElement(atomic),
+      onClick: undefined,
+    },
+  };
+}
+
+async function renderTable() {
+  render(<PeriodicTable elementRenderer={elementRenderer} />);
+  await waitForElementToBeRemoved(
+    () => screen.queryAllByLabelText(/loading/i),
+    { timeout: 4000 }
+  );
+}
```

Comment:
I personally don't see much point in having this function extracted out of the test when it's only used in one and the other has it's own render logic.  Consider inlining it into the tests.